src/SampleLibraryUI/SampleLibraryUI.csproj

```diff
@@ -199,20 +20,14 @@
   <ItemGroup>
     <Content Include="images\logo.png" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <PropertyGroup>
-    <PostBuildEvent>
-    </PostBuildEvent>
-  </PropertyGroup>
-  <PropertyGroup>
-    <PreBuildEvent>
-    </PreBuildEvent>
-  </PropertyGroup>
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
+  <ItemGroup>
+    <PackageReference Include="DynamoVisualProgramming.DynamoServices" Version="3.0.0-beta7064" />
+    <PackageReference Include="DynamoVisualProgramming.WpfUILibrary" Version="3.0.0-beta7064" />
+    <PackageReference Include="DynamoVisualProgramming.ZeroTouchLibrary" Version="3.0.0-beta7064" />
+  </ItemGroup>
+  <Target Name="AfterBuildOps" AfterTargets="Build">
     <ItemGroup>
-      <PackageFiles Include="$(OutDir)\SampleLibraryUI.dll;$(OutDir)\SampleLibraryUI.XML;$(OutDir)\SampleLibraryUI.dll.config" />
+      <PackageFiles Include="$(OutDir)\SampleLibraryUI.dll;$(OutDir)\SampleLibraryUI.XML" />
       <ENResourceFiles Include="$(OutDir)\en-US\*.*" />
       <ESResourceFiles Include="$(OutDir)\es-ES\*.*" />
     </ItemGroup>

```

Comment:
For some reason I'm having difficulty getting these afterbuild targets to run in Meshtoolkit